packages/jsii/bin/jsii.ts

```diff
@@ -54,6 +54,12 @@ const warningTypes = Object.keys(enabledWarnings);
             desc: `List of warnings to silence (warnings: ${warningTypes.join(
               ',',
             )})`,
+          })
+          .option('strip-deprecated', {
```

Comment:
Expand and document this in the jsii docs. Also explain any pre-requisites, such as deprecated types must not be used in such-and-such places, etc.

Reply:
The whole point of it being `EXPERIMENTAL` here is I actually don't want to publicise it too much just yet...

Reply:
Maybe add an experimental section to the docs 🤷‍♂️
Fine with me if you feel strongly